pkg/kubectl/resource_printer.go

```diff
@@ -270,10 +268,11 @@ func formatEndpoints(endpoints []api.Endpoint) string {
 		return "<none>"
 	}
 	list := []string{}
-	for i := range endpoints {
-		ep := &endpoints[i]
-		list = append(list, net.JoinHostPort(ep.IP, strconv.Itoa(ep.Port)))
-	}
+	//FIXME: What do we want to print, now that endpoints are more complex?
```

Comment:
Does a simple comma-separated list not work? This is only called in `kubectl describe`, so it can be long/verbose or even use newlines.